packages/shared/Messages.ts

```diff
@@ -17,15 +17,6 @@
  ***********************************************************************/
 
 export enum Messages {
-  /**
```

Comment:
Seems unrelated ?

Reply:
There are messages which where used in the Playground v1, there are now useless, as shown by the typecheck not raising any error (my editor was telling me there were unused also)